aten/src/ATen/native/cpu/UpSampleKernel.cpp

```diff
@@ -1376,8 +1377,22 @@ void upsample_bilinear2d_aa_kernel_impl(
     c10::optional<double> scales_h,
     c10::optional<double> scales_w) {
 
-  separable_upsample_generic_Nd_kernel_impl<2, scale_t, HelperInterpLinear>(
-      output, input, align_corners, {scales_h, scales_w});
+  if (input.dtype() == at::kByte) {
```

Comment:
you might want to use something like `_use_vectorized_kernel_cond` here (a few functions in the file seem to be using that condition to decide whether or not to vectorize)

Reply:
synced offline -- currently the focus is to only support this for uint8. In the future, we can extend this to other data types.